src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -973,7 +973,10 @@ class Config:
     async def __validate(self, verrors, data, schema_name, old=None):
 
         if not RE_TARGET_NAME.search(data['name']):
-            verrors.add(f'{schema_name}.name', 'Use alphanumeric characters, ".", "-" and ":".')
+            verrors.add(
+                f'{schema_name}.name',
+                'Alphanumeric with lowercase characters or these characters ".", "-" and ":" are allowed.'
```

Comment:
"With" generally is read as "in addition to".  So this implies that uppercase can still be used. Using quotes on the allowed characters makes it harder to read.  Suggestion:

`` Lowercase alphanumeric characters plus dot (.), dash (-), and colon (:) are allowed. ``